test/comment_spec.rb

```diff
@@ -0,0 +1,7 @@
+BDDSM.describe 'Test case with comments' do
```

Comment:
This test will  fail so, build will be red after #34 is merged.

Reply:
I will fetch it after merge and retest this again.

Reply:
merged